### PR TITLE
Update script and automation with the Server backend

### PR DESCRIPTION
**Pull request by Tomcli**
This PR should make the *Deploy to IBM Cloud button* able to create the visual recognition service, train the model, and push the server to use the trained model. I also updated the instructions to use the command line to upload the custom classifier. Now, the backend portion should be done and all we need is the mobile app and possibly more pictures for the custom model.

### CODE REVIEW

File: server/custom-model.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+cf create-service-key visual-recognition-wws waste-sorter
+API_KEY=$(cf service-key visual-recognition-wws waste-sorter | awk ' /api_key/ {print $2;exit}' | tr -d "\",")
```
Comment (stevemar) [inline server/custom-model.sh]: you can use the --guid option here

File: server/custom-model.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+if [ ${#API_KEY} -eq 0 ]; then
+	echo 'Cannot retrieve your visual recognition api-key, exiting now.'
+	exit 1
+fi
+curl -X POST -F "Landfill_positive_examples=@resources/Landfill.zip" -F "Recycle_positive_examples=@resources/Recycle.zip" -F "Compost_positive_examples=@resources/Compost.zip" -F "negative_examples=@resources/Negative.zip" -F "name=waste" "https://gateway-a.watsonplatform.net/visual-recognition/api/v3/classifiers?api_key=$API_KEY&version=2016-05-20"
```
Comment (stevemar) [inline server/custom-model.sh]: minor suggestion for another patch: the files should be lowercase to avoid case issues on windows/linux/mac platforms

**Comment by Tomcli** [comment]
@stevemart Thanks for the review.

**Review by stevemar — APPROVED** [review]
thanks for making the changes @Tomcli

**Pull request merged**